### DISCUSSION TITLE
Auto-Adjusting server/task hours with Daylight Savings

### DISCUSSION THREAD

**Discussion by Decollates** [gameserverapp/Platform#963] [Suggestions]
Currently, with daylight savings, timers will be delayed by an hour from when they're on normal time. As an example, a server set to restart at midnight/0000 will now restart at 1am/0100. For servers that want to keep these times the same throughout the year, I am requesting the feature to automatically update the hours to reflect daylight savings time changes, with the option to disable the update for users that do not utilize daylight savings.

**Comment by mvaessen** [comment]
Thanks for your suggestion, I assume with `with the option to disable the update for users that do not utilize daylight savings` you're referring to GSA users, right?

It's kind of complicated, since all team members on a single dashboard can have different timezones. So in that case GSA would need to adjust the timer based on the owner's timezone.

Also note that daylight saving is not synced across the globe: https://www.quora.com/Does-daylight-saving-time-flip-on-the-same-day-across-the-world
